pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -490,6 +490,33 @@ type SNIIngress struct {
 
 // ETCDConfig contains ETCD related configs
 type ETCDConfig struct {
+	// ETCDController contains config specific to ETCD controller
+	ETCDController *ETCDController `json:"etcdController,omitempty"`
+	// CustodianController contains config specific to custodian controller
+	CustodianController *CustodianController `json:"custodianController,omitempty"`
+	// BackupCompactionController contains config specific to backup compaction controller
+	BackupCompactionController *BackupCompactionController `json:"backupCompactionController,omitempty"`
```

Comment:
```suggestion
	// ETCDController contains config specific to ETCD controller
	// +optional
	ETCDController *ETCDController `json:"etcdController,omitempty"`
	// CustodianController contains config specific to custodian controller
	// +optional
	CustodianController *CustodianController `json:"custodianController,omitempty"`
	// BackupCompactionController contains config specific to backup compaction controller
	// +optional
	BackupCompactionController *BackupCompactionController `json:"backupCompactionController,omitempty"`
```

Reply:
I don't understand. You suggested here to add `optional` tag. Right?

Reply:
This is `v1alpha1`, so here we want to have `optional` and `json` tags. In internal APIs we don't.

Reply:
I see. Thanks for explanation

Reply:
I made the changes.. Sorry for misunderstanding